woodwork/column_accessor.py

```diff
@@ -357,6 +361,62 @@ def set_semantic_tags(self, semantic_tags):
             _raise_init_error()
         self._schema._set_semantic_tags(semantic_tags)
 
+    def outliers_dict(self, low_bound=None, high_bound=None):
+        """Gets the values of a series that lay above and below specified bounds.
+
+        Args:
+            series (Series): Data for which the outliers should be determined.
+            low_bound (float, optional): The number below which outliers lay. Is inclusive.
+            high_bound (float, optional): The number above which outliers lay. Is inclusive.
+
+        Note:
+            If neither or only one of low_bound or high_bound is passed in, the bounds will be calculated
+            using the IQR method.
+
+        Returns:
+            dict[str -> float,list[number]]: a dictionary containing outlier values and their corresponding indexes.
+                The following elements will be found in the dictionary:
+
+                - low_values (list[float, int]): the values of the lower outliers
+                - high_values (list[float, int]): the values of the upper outliers
+                - low_indices (list[int]): the corresponding index values for each of the lower outliers
+                - high_indices (list[int]): the corresponding index values for each of the upper outliers
+        """
+        if self._schema is None:
+            _raise_init_error()
```

Comment:
Yeah, it goes in front of every method on the column and table accessors to avoid the unexpected errors that can show up when you try and make Woodwork calls before initializing Woodwork. 

I don't think we've ever considered adding decorators, though I do think this could be a good use for that. @simha104 is working on an issue that could end up touching a lot of these lines in order to add schema validation, so that might be a good time to change all of these into decorators! (@thehomebrewnerd any thoughts?)

Reply:
I think the decorator approach is better than what we are doing currently, and I like that idea.

If we could auto-magically have this applied to our methods somehow that might be even better so we don't always have to remember to add the code/decorator to new methods that get added. That might be more problematic/difficult than it's worth though...

Reply:
@thehomebrewnerd Yeah, in principle it's nice to avoid the possibility that someone might fail to add the magic decorator to a new method.  In practice, I always have concerns that automagical meta class stuff is hard to understand and predict for most developers.  So I often seek some kind of middle ground.  I think the decorator thing is nice because you can call it something like `require_init` and that's pretty descriptive and hard to miss.  The trade-off is that you have to remember to include it.  In any case, we'd need some way to distinguish methods that should have that behavior from those that shouldn't.  That could require some other machinery which might be just as much code as the simple decorator application.